util/test_base_cuda.cpp

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Copyright:	(c) 2017 by Codeplay Software LTD. All Rights Reserved.
+//
+*******************************************************************************/
+
+#include "test_base_cuda.h"
+#include "../tests/common/cts_selector.h"
+#include "../tests/common/get_cts_object.h"
+#include "../tests/common/macros.h"
+
+#ifdef _MSC_VER
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+#ifdef SYCL_BACKEND_CUDA
+// conformance test suite namespace
+namespace sycl_cts {
+namespace util {
```

Comment:
```suggestion
namespace sycl_cts::util {
```

Reply:
This file has been removed with the change to Catch2